css-positioning/relative.css

```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  top: 20px;
+}
+
+#charmander {
+  position: relative;
+  top: 35px;
+  /* your code here */
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 70px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 155px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 300px;
+}
```

Comment:
Instead of applying `position: relative` over and over, you should apply it once in the `.pokemon > img` rule. Less code, same effect!